app/src/main/java/com/example/cm/Constants.java

```diff
@@ -39,7 +39,7 @@
     public static final float FINAL_CARD_ALPHA = 1f;
     public static final float ON_SNAPPED_MAP_ZOOM = 15.0f;
     public static final int DEFAULT_CARD_OFFSET = 0;
-    public static final int MAX_CLUSTER_ITEM_DISTANCE = 15;
+    public static final int MAX_CLUSTER_ITEM_DISTANCE = 40;
     public static final int CURRENT_USER_Z_INDEX = 999;
```

Comment:
why is CURRENT_USER_Z_INDES 999 and MEETUP_Z_INDEX 9999? do we need two values for that?

Reply:
Meetups should always cluster above user clusters hence a second Z_INDEX is needed here